### ISSUE TITLE
Reorder bank accounts

### ISSUE THREAD

**Issue by mollwe** [liato/android-bankdroid#119]
Right now banks is displayed in the order they were added. A feature fore reorder them would be nice.

**Comment by fredrike** [comment]
@fredriklindberg please create a pull request..

**Comment by fredrike** [comment]
I really like this idea. However, as [cwac-touchlist](https://github.com/commonsguy/cwac-touchlist) is  discontinued, another lib should be used.

**Comment by goober** [comment]
I agree with you. However, I think we should wait to implement this feature until v2 so we don´t need to do it all over again.

**Comment by fredriklindberg** [comment]
That commit bit too old and outdated to create a pull request :)
I don't use bankdroid these days, but feel free to use parts/ideas from my commit if you find it useful.
